yarn-project/types/src/interfaces/hasher.ts

```diff
@@ -3,27 +3,27 @@
  */
 export interface Hasher {
   /**
-   * Compresses two 32-byte hashes.
-   * @param lhs - The first hash.
-   * @param rhs - The second hash.
+   * Hash two 32-byte hashes.
+   * @param lhs - The first 32-byte array.
+   * @param rhs - The second 32-byte array.
```

Comment:
I'll change this -- theres nothing here forcing the lhs and rhs to be 32 bytes.

Also: 

+ Hash two arrays
- Hash two 32-byte hashes.

Reply:
The return value also does not need to be 32 bytes but this was there before, so I'll leave that as is